pkg/controller/virtinformers.go

```diff
@@ -167,6 +172,15 @@ func (f *kubeInformerFactory) OfflineVirtualMachine() cache.SharedIndexInformer
 	})
 }
 
+func (f *kubeInformerFactory) ConfigMap() cache.SharedIndexInformer {
+	// We currently only monitor configmaps in the kube-system namespace
+	return f.getInformer("configMapInformer", func() cache.SharedIndexInformer {
+		restClient := f.clientSet.CoreV1().RESTClient()
+		lw := cache.NewListWatchFromClient(restClient, "configmaps", systemNamespace, fields.Everything())
```

Comment:
You should not watch every configmap. Use `fields.OneTermEqualSelector("metadata.name", "kubevirt-config")`. That will work.

Reply:
That makes sense. Nothing from any other ConfigMap will ever be a concern to us. Good call!